workflow/rules/common.smk

```diff
@@ -23,9 +23,15 @@ def read_sample_config(config_f):
 
             samples_dict[sample] = {"vcf": vcf, "bam": bam}
 
+            # expect only filepath per field
             for colname in ["capture_bed"]:
                 if colname in row:
                     samples_dict[sample][colname] = row[colname]
```

Comment:
```suggestion
            if "capture_bed" in row:
                samples_dict[sample][colname] = row[colname]
```
for loop not needed like other comment

Reply:
never mind on this, I saw in #90 that this is setup for a larger structure that just isn't realized in the PR